### DISCUSSION TITLE
How to turn off the function of automatically creating mysql tables when the service starts

### DISCUSSION THREAD

**Discussion by handong0123** [milvus-io/milvus#4635] [Q&A and General discussion]
In the production environment, the mysql account does not have ddl permissions,  DBA can create the table.
So I created all the tables in advance, but I still report that there is no permission to create the table when I start.

**Comment by shiyu22** [comment]
Hi, this feature is not supported, you can submit us an [feature issue](https://github.com/milvus-io/milvus/issues/new/choose).
